cmd/zetaclientd/init.go

```diff
@@ -1,13 +1,25 @@
 package main
 
 import (
+	"encoding/json"
+	"os"
+	"path"
+
 	"github.com/rs/zerolog"
 	"github.com/spf13/cobra"
 
 	"github.com/zeta-chain/zetacore/zetaclient/config"
 	"github.com/zeta-chain/zetacore/zetaclient/testutils"
 )
 
+// solanaTestKey is a local test private key for Solana
+var solanaTestKey = []uint8{
```

Comment:
Is it the standard format for key? Can't we have a string directly?

Reply:
It's the file format used by Anchor Go environment (similar to Hardhat in Ethereum).